src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <div>
+      <label>{{selectLabel}}<span v-if=isRequired>*</span></label>
+    </div>
+    <div>
+      <select
+        v-bind:aria-label="selectLabel"
```

Comment:
This should be associated with the label rather than containing the same text in the aria-label. We should give the label a unique id and use aria-labelledby on the corresponding element.